config.json

```diff
@@ -1,4 +1,4 @@
-{
```

Comment:
I am seeing `Unexpected UTF-8 BOM (decode using utf-8-sig): line 1 column 1 (char 0)` after this line was change. Can you please revert this change?